### ISSUE TITLE
question about 'vvc_competition' loss

### ISSUE THREAD

**Issue by wl3b10s** [bbc/cnn-fractional-motion-compensation#2]
'vvc_competition' achieves good improvement for it ignores 'not' good result for model update.

but does it a good choice during epoch validation?

i guess competition may result in different final output channels are used for calculating validation loss with same validation data.
then is it fair to compare loss that may result from different output channels ?

thank you. @lmurn

**Comment by lmurn** [comment]
Hi Liang,

You raise a good point. During network training, we only update the network for the very best channel for each input instance. During validation, we only retain the loss from the best performing channel (and compare it to VVC loss) and ignore other channels. I have experienced some issues with validation losses before, so it might be a good idea to investigate whether other channels contain results that may inform our results and performance. 

Apologies if I have misunderstood any part of your question. I'll leave this issue open for further comments.
